core/Pos/Core/Block/Blockchain.hs

```diff
@@ -1,5 +1,13 @@
 {-# LANGUAGE Rank2Types   #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE DefaultSignatures #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE OverloadedStrings   #-}
```

Comment:
If you take `.` out of the source-dirs in the cabal file, then these extension pragmas shouldn't be necessary.

Style nit: `stylish-haskell` should align these.